mathics/builtin/arithmetic.py

```diff
@@ -35,14 +35,11 @@
     from_mpmath,
     from_python,
 )
-from mathics.core.symbols import Symbol
+from mathics.core.symbols import Symbol, SymbolFalse, SymbolList, SymbolTrue
 from mathics.core.systemsymbols import (
```

Comment:
I suspect those __init__ imports were allowing incorrect imports like this from occurring.

Reply:
OK, I do not have clear what is the best way to do this. I thought that import Symbol* from systemsymbols was more consistent and easy to change/update in the future (let's say, using ed). To change all the modules to load symbols from their new place would have been easier if the different symbols had been in different import lines.